_rules/html-xml-lang-match-5b7ae0.md

```diff
@@ -27,11 +27,9 @@ The root element of the [web page](https://www.w3.org/TR/WCAG21/#dfn-web-page-s)
 
 For each test target, the value of the primary language subtag (characters before the first dash) for the `lang` and `xml:lang` attributes are the same.
 
-**Note**: HTML 5 specification requires the `lang` and `xml:lang` attributes to match exactly. This is not known to impact accessibility, which is why it is permitted in this rule.
-
 ## Assumptions
 
-- Although there is no known way that an inappropriate secondary subtag in the `lang` or `xml:lang` attribute (such as "en-XYZ") can lead to accessibility issues, it is conceivable that with specific assistive technologies on some languages, there can be exceptions. In that case `lang` and `xml:lang` should have matching secondary subtags.
+- Although there is no known way that an inappropriate secondary subtag in the `lang` or `xml:lang` attribute (such as `en-XYZ`) can lead to accessibility issues, it is conceivable that with specific assistive technologies on some languages, there can be exceptions. In that case `lang` and `xml:lang` should have matching secondary subtags.
```

Comment:
Secondary subtags are important for some languages - and the wrong subtag results in voicing that is hard (or impossible) to understand:
https://github.com/act-rules/act-rules.github.io/issues/668#issuecomment-534465673

Reply:
Any suggestions on how to phrase the assumption to account for this?

Reply:
Chatted to @nitedog  and @WilcoFiers about this - there's a W3 Pronunciation Task force that might have some useful input on this https://www.w3.org/WAI/APA/task-forces/pronunciation/

Reply:
Here's an example that shows the problem with ignoring subtags - these two code fragments specify exactly the same languages  (because lang=zh-yue and lang=yue are aliases that identify the same language) but only one of the examples fails the rule. 

See https://tools.ietf.org/html/bcp47#section-2.2.2

```
<!-- this fails the rule because primary tags don't match -->
<html lang='yue' xml:lang='cmn' >
```

```
<!-- this passes the rule because primary tags match -->
<html lang='zh-yue' xml:lang='zh-cmn' >
```

Reply:
That's exactly the kind of argument I needed 💯 Cheers!

Reply:
The rule now requires both the primary and the extended language subtags to match. Is there a similar argument for requiring script and/or other subtags to match as well? Are these even used by UAs and ATs?

Reply:
See: https://github.com/act-rules/act-rules.github.io/pull/934#discussion_r328105044